kafka-consumer/main.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import threading
+
+import dispatch
+import requests
+from kafka import KafkaConsumer
+
+
+BROKERS = ["localhost:9092", "localhost:9093", "localhost:9094"]
+TOPIC = "topic1"
+GROUP = "group1"
+
+
+@dispatch.function
+def send_message(url, payload):
+    response = requests.post(url, data=payload)
+    response.raise_for_status()
+    return response
+
+
+def forward_messages(url):
+    consumer = KafkaConsumer(TOPIC, group_id=GROUP, bootstrap_servers=BROKERS)
+    try:
+        for message in consumer:
+            send_message.dispatch(url, message.value)
```

Comment:
I'll see if there's a lower level API we can use to fetch batches of messages. That way we can send batches of function calls to Dispatch, and control the timeout when fetching a batch of messages from Kafka.